gordo/workflow/workflow_generator/resources/argo-workflow.yml.template

```diff
@@ -451,9 +451,9 @@ spec:
                   apiVersion: v1
                   kind: Service
                   metadata:
-                    name: gordo-postgres-{{project_name}}
+                    name: {{postgres_host}}
```

Comment:
Could we possibly ever want to have multiple postgres hosts? Maybe if we were to change the schema and do a canary, blue/green deployment or something? No change request here... just (possibly dumb) discussion.

Reply:
Again, I think that is outside the scope of what this PR is doing. I think it's an entirely valid consideration, however.